packages/kanki/kanki/main.js

```
@@ -0,0 +1,1113 @@
+    
+    // Reposition any visible popups or toasts
+    var toast = document.getElementById("toastNotification");
+    if (toast && toast.style.display === "block") {
+      var screenHeight = window.innerHeight || document.documentElement.clientHeight || document.body.clientHeight;
+      toast.style.top = (screenHeight > 1000) ? "120px" : "80px";
+    }
+    
+    var overlay = document.getElementById("confirmationOverlay");
+    if (overlay && overlay.style.display === "block") {
+      var popup = overlay.querySelector(".popup");
+      var screenHeight = window.innerHeight || document.documentElement.clientHeight || document.body.clientHeight;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'screenHeight'. Consider to delete it or rename it.

'screenHeight' is defined here:


(lint/suspicious/noRedeclare)